packages/ui5-application-writer/src/types.ts

```diff
@@ -9,6 +10,7 @@ export interface Package {
     ui5?: {
         dependencies?: string[];
     };
+    sapuxLayer?: UI5FlexLayer;
```

Comment:
👍

Reply:
Would you please add a comment to the property explaining its use and default assignment.